Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+#Basic copy cat design of the mood file.
+#Figured I shouldn't fix something that isn't broken.
+#Shout out to whoever did it right the first time.
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imhaving",category=['you'],prompt="I'm Having",pool=True,unlocked=True))
+label monika_imhaving:
+    m 1eub "You know, although I don't get hungry, I still miss enjoying good food."
+    m 4eub "Like Natsuki's cupcakes."
+    m 4eub "For a bunch of lines of code, they were pretty tasty."
+    m 3hub "Are you eating anything right now, [player]?"
+
+
+default persistent_.mas_food_database = {}
+
+default persistent_.mas_food_current = None
+
+init 1 python in mas_food:
+    food_db = dict()
+
+    TYPE_FOOD = 0
+    TYPE_DRINK = 1
+
+    FOOD_X = 680
+    FOOD_Y = 40
+    FOOD_W = 560
+    FOOD_H = 640
+    FOOD_XALIGN = -0.05
+    FOOD_AREA = (FOOD_X, FOOD_Y, FOOD_W, FOOD_H)
+    FOOD_RETURN = "Nothing"
+
+
+    def getFoodType(food_label):
+
+        food = food_db.get(food_label)
+
+        if food:
+            return food.category[0]
+
+        return None
+
+label mas_food_start:
+    python:
+
+        import store.mas_food as mas_food
+
+
+        filtered_food= Event.filterEvents(
+            mas_food.food_db,
+            unlocked=True,
+            aff=mas_curr_affection
+            )
+
+        food_menu_items = [
+        (mas_food.food_db[k].prompt, k, False, False)
+        for k in filtered_food
+        ]
+
+        food_menu_items.sort()
+
+        final_item = (mas_food.FOOD_RETURN, False, False, False, 20)
+    call screen mas_gen_scrollable_menu(food_menu_items, mas_food.FOOD_AREA, mas_food.FOOD_XALIGN, final_item=final_item)
+    if _return:
+        $ pushEvent(_return)
+
+        $ persistent._mas_food_current = _return
+    return _return
+
+
+
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_coffee',prompt='Coffee',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+
+label mas_food_coffee:
+    m 5hua "That sounds wonderful!"
+    m 5eua "I really miss having coffee."
+    m 3eub "I would always have a cup before and after school."
+    m 3hub "Maybe even sometimes during class."
+    m 2euc "I wish I could figure out a way to get some in here."
+    m 2hub "Who knows, maybe you could share a cup with me, [player]"
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_water',prompt='Water',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_water:
+    m 1eua "I'm really happy to hear that [player]!"
+    m 1eub "Water is probably the healthiest thing that you could drink."
+    m 3eua "Seeing how we can only live for a few days without it."
+    m 3eud "Just promise you get plenty to drink each day."
+    m 2lud "I don't want you getting sick because you didn't drink any."
+    m 2eua "Even if it's just a glass or two, do it for me."
+    return
+init 5 python:
```

Comment:
For ease of reading and consistency, it would help to add a line of whitespace between each topic.